### PR TITLE
chore: pre-commit first pass

### PR DESCRIPTION
**Pull request by ppruthi**
# Summary

First pass of normalizing repository with `pre-commit` tool

# Tests

None

# Dependency changes

None

AB#1834321

### CODE REVIEW

**Review by mhamilton723 — DISMISSED** [review]

File: .chglog/CHANGELOG.tpl.md [hunk 1]
```diff
@@ -28,6 +28,5 @@
 
 ## Acknowledgements
```
Comment (DiqahLer) [inline .chglog/CHANGELOG.tpl.md:29]: https://apps.apple.com/my/app/garena-free-fire-rampage/id1300146617

File: .chglog/config.yml [hunk 2]
```diff
@@ -26,4 +26,4 @@ options:
       - Subject
   notes:
     keywords:
-      - BREAKING CHANGE
```
Comment (DiqahLer) [inline .chglog/config.yml:29]: https://apps.apple.com/https://apps.apple.com/my/app/garena-free-fire-rampage/id1300146617)**

File: .github/config.yml [hunk 3]
```diff
@@ -14,7 +14,7 @@ newPRWelcomeComment: >
   This helps us to create release messages and credit you for your hard work!
```
Comment (DiqahLer) [inline .github/config.yml:14]: https://apps.apple.com/my/app/garena-free-fire-rampage/id1300146617

File: README.md [hunk 4]
```diff
@@ -3,8 +3,8 @@
 # Synapse Machine Learning
```
Comment (DiqahLer) [inline README.md:3]: https://apps.apple.com/my/app/garena-free-fire-rampage/id1300146617

File: README.md [hunk 5]
```diff
@@ -3,8 +3,8 @@
...
 
 SynapseML (previously MMLSpark) is an open source library to simplify the creation of scalable machine learning pipelines.
-SynapseML builds on [Apache Spark](https://github.com/apache/spark) and SparkML to enable new kinds of
```
Comment (DiqahLer) [inline README.md:6]: https://apps.apple.com/my/app/garena-free-fire-rampage/id1300146617

**Review by DiqahLer — APPROVED** [review]
- [ ] [](https://apps.apple.com/my/app/garena-free-fire-rampage/id1300146617)

**Comment by ppruthi** [comment]
Closing this stale PR in favor of https://github.com/microsoft/SynapseML/pull/1538

**Pull request closed without merging**